unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -6936,109 +6926,184 @@ public bool objectIsCurrentlyVisible(SimObjPhysics sop, float maxDistance)
             }
 
             return allVisible;
-
         }
 
-        public void PositionsFromWhichItemIsInteractable(ServerAction action) {
+        // returns null on failure.
+        // @positions/@rotations/@horizons/@standings are used to override all possible values the agent
+        // may encounter with basic agent navigation commands (excluding teleport).
+        private List<Dictionary<string, object>> getInteractablePoses(
```

Comment:
We are now returning `List<Dictionary<string, object>` since that is easiest to use with python. Here, the new syntax is:
```python
poses = controller.step('GetInteractablePoses', objectId=<>).metadata['actionReturn']
controller.step('TeleportFull', **poses[0])
```
instead of the old long equivalent
```python
poses = controller.step('PositionsFromWhichItemIsInteractable', objectId=<>).metadata['actionReturn']
i = 0
controller.step('TeleportFull',
    x=poses[i]['x'], y=poses[i]['y'], z=poses[i]['z'],
    horizon=poses[i]['horizon'],
    standing=bool(poses[i]['standing']),
    rotation=poses[i]['rotation']
)
```
Note that the latter still works for backward compatibility, but is deprecated!